### PR TITLE
refactor: restructure app + frontend

### PR DESCRIPTION
**Pull request by maschad**
I've made some UI adjustments  to help make it easier to debug deploying liquidity.

I've also restructured the frontend of the app.

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "build": "fuels build",
+    "lint": "eslint .",
+    "fuels:dev": "fuels dev",
+    "test": "vitest"
+  },
+  "dependencies": {
+    "fuels": "^0.97.2"
+  },
+  "devDependencies": {
+    "dotenv": "^16.4.7",
```
Comment (matt-user) [inline app/package.json:19]: should eslint be a dev-dep?

File: app/pnpm-workspace.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+packages:
+  - "frontend/*"
```
Comment (matt-user) [inline app/pnpm-workspace.yaml:2]:
```suggestion
  - "frontend/"
```

File: app/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "strip-",
+  "version": "1.0.0",
+  "description": "A simple p2p casino",
+  "workspaces": [
+    "frontend/*",
+    "scripts/*"
+  ],
+  "scripts": {
+    "build": "fuels build",
+    "lint": "eslint .",
```
Comment (matt-user) [inline app/package.json:11]: the lint command does not work in the app folder

**Review by matt-user — APPROVED** [review]
looks good, just some small stuff

**Comment by AurelienFT** [comment]
Push to make it common and start changes set on empty changes.

**Pull request merged**